aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -61,5 +61,9 @@ module aptos_framework::aggregator_v2 {
 
     public native fun snapshot<Element>(aggregator: &Aggregator<Element>): AggregatorSnapshot<Element>;
 
+    public native fun create_snapshot<Element: copy + drop>(value: Element): AggregatorSnapshot<Element>;
+
+    public native fun copy_snapshot<Element: copy + drop>(snapshot: &AggregatorSnapshot<Element>): AggregatorSnapshot<Element>;
```

Comment:
Do we need this for Token V2? Are we copying snapshots anywhere, or are we only moving snapshots in Token v2?